lib/src/config.dart

```diff
@@ -196,20 +200,24 @@ void _parseDeletionPatterns(yaml, GlobalConfig config) {
       .addAll((patterns as YamlList).map((pattern) => new RegExp(pattern)));
 }
 
-void _parseFilesToLoad(yaml, GlobalConfig config) {
+// TODO : USE package resolver to resolve these
+Future _parseFilesToLoad(yaml, GlobalConfig config) async {
   var filePaths = _parseStringList(yaml, 'files_to_load');
   if (filePaths == null) return;
-  config.filesToLoad.addAll(filePaths.map((filePath) {
+  config.filesToLoad.addAll(await Future.wait(filePaths.map((filePath) async {
     var parts = filePath.split(':');
     if (parts.length == 1) {
       return platformIndependentPath(parts[0]);
     } else if (parts.length == 2 && parts[0] == 'package') {
-      return path.join('packages', platformIndependentPath(parts[1]));
+      Uri uri = await config.packageResolver.resolveUri(filePath);
+      String p = platformIndependentPath(uri.toFilePath());
+      p = p.replaceAll("%", r'%25'); // Reset url encoding
```

Comment:
I've not gone in deep, but somewhere after that point the path got its "%XX" replaced with hex char with code `XX`. 

That was causing a "FileNotFound" exception.